server/repository/src/db_diesel/name_property_row.rs

```diff
@@ -0,0 +1,126 @@
+use super::{name_property_row::name_property::dsl::*, property_row::property};
+
+use serde::{Deserialize, Serialize};
+
+use crate::ChangeLogInsertRow;
+use crate::ChangelogRepository;
+use crate::ChangelogTableName;
+use crate::RepositoryError;
+use crate::RowActionType;
+use crate::StorageConnection;
+use crate::Upsert;
+
+use diesel::prelude::*;
+
+table! {
+    name_property (id) {
+        id -> Text,
+        property_id -> Text,
+    }
+}
+joinable!(name_property -> property (property_id));
+// TODO: I shouldn't need this??
+allow_tables_to_appear_in_same_query!(name_property, property);
```

Comment:
Send help? 

My join query wouldn't compile without this, which makes me think joinable! above isn't doing something right, but I can't figure out what?

Reply:
I think this is expected. Joinable, is kind of a shortcut macro to avoid putting in an explicit `on` clause.
Docs say you need to call both here.
https://docs.rs/diesel/latest/diesel/macro.joinable.html

Reply:
Ah thanks James - I missed the `diesel_schema.rs` file where we have a lot of these allow in same query definitions set up, and couldn't figure out why some tables needed it and some didn't 😅